src/test_l1_tvSettings.c

```diff
@@ -15902,11 +15902,13 @@ void test_l1_tvSettings_positive_GetTVGammaTarget (void)
     {
         snprintf(keyValue, UT_KVP_MAX_ELEMENT_SIZE, "tvSettings/ColorTemperature/index/%d", i);
         colorTempValue = (tvColorTemp_t) UT_KVP_PROFILE_GET_UINT32(keyValue);
-        GetTVGammaTarget( colorTempValue, &x_Value, &y_Value);
+        result = GetTVGammaTarget( colorTempValue, &x_Value, &y_Value);
+        UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
         UT_ASSERT_TRUE((x_Value >= 0 && x_Value <= 1.0) );
         UT_ASSERT_TRUE((y_Value >= 0 && y_Value <= 1.0) );
 
-        GetTVGammaTarget( colorTempValue, &x_ValueRetry, &y_ValueRetry);
+        result = GetTVGammaTarget( colorTempValue, &x_ValueRetry, &y_ValueRetry);
+        UT_ASSERT_EQUAL_FATAL(result, tvERROR_NONE);
```

Comment:
Dont use UT_ASSERT_EQUAL_FATAL instead use UT_ASSERT_EQUAL

Reply:
addressed in commit id:- 45ba5edef84a528bb3ef3e891c40bc01d1646c0a